.travis.yml

```diff
@@ -48,9 +48,18 @@ script:
 - diff -q <(travis-worker --help) <(travis-worker -h)
 - travis-worker --echo-config
 - travis-worker --list-backend-providers
-- mkdir -p bin
 - cp -v ~/gopath/bin/travis-worker ./bin/
 - make package
+- ./bin/travis-worker-install --help
+- sudo ./bin/travis-worker-install --trace --skip-docker-populate --packagecloud-source=travisci/worker-testing
```

Comment:
@danishkhan the option that will pull from the "edge" repo is `--packagecloud-source=travisci/worker-testing` :smiley_cat: